tests/test_association_task.py

```diff
@@ -75,13 +95,40 @@ def test_run(self):
                 [0]):
             self.assertEqual(test_obj_id, expected_obj_id)
 
+    def test_run_trailed_sources(self):
+        """Test the full task by associating a set of diaSources to
+        existing diaObjects when trailed sources are filtered
+        # """
+
+        config = AssociationTask.ConfigClass()
+        config.doLongTrailFilter = True
+        assocTask = AssociationTask(config=config)
+
+        results = assocTask.run(self.diaSources, self.diaObjects, self.exposure)
+
+        print(results.nUpdatedDiaObjects, "results nupdated")
+        print(len(self.diaObjects) - 3, " length of dia objects")
```

Comment:
No print statements in task or test code.

Reply:
Yup, forgot to remove that after testing. Removed now.